docs/configuration/downstream/pull_from_upstream.md

```diff
@@ -9,7 +9,15 @@ A dist-git only job that opens a new dist-git pull request in
 src.fedoraproject.org when a new upstream release happens using a notification
 from [release-monitoring.org](https://release-monitoring.org/). Upstream Release Monitoring creates a Bugzilla
 for the new upstream release and this Bugzilla is by default referenced in the dist-git changelog as
-`- Resolves rhbz#xz` and `- Resolves rhbz#xz` in the commit message. This behaviour can be customized
+`- Resolves rhbz#xz` and `- Resolves rhbz#xz` in the commit message. 
+
+The `pull_from_upstream` job will close the Bugzillas opened by the Upstream Release Monitoring on your behalf 
+through the [Bodhi updates](https://fedora-infra.github.io/bodhi/6.0/user/automatic_updates.html#associate-bugs-to-automatic-updates). 
+There is also an env variable with the list of bugs to be closed
+`PACKIT_RESOLVED_BUGS` that you can use in the case you want to customize the changelog creation but still have
+the bugs automatically closed.
+
+This job behaviour can be customized
 using [actions](/docs/configuration/actions#syncing-the-release).
```

Comment:
I would mention that the env variable is exposed for `commit-message` and `changelog-entry` actions